instaloader/instaloader.py

```diff
@@ -150,6 +151,7 @@ class Instaloader:
 
     def __init__(self,
                  sleep: bool = True, quiet: bool = False,
+                 request_timeout: Optional[int] = None,
```

Comment:
There is an invocation of the Instaloader constructor in Instaloader.anonymous_copy() (line 196+) which lacks the request_timeout argument. Please add `request_timeout=self.context.request_timeout` there.